src/main/webapp/app/lecture/lecture-unit/lecture-unit-management/attachment-units/attachment-units.component.html

```diff
@@ -78,21 +78,34 @@ <h2 id="page-heading">
         <input class="form-check-input" type="checkbox" name="processUnit" [disabled]="units.length < 1" (change)="onSelectRemoveBreakSlides()" />
         <span class="px-1">{{ 'artemisApp.attachmentUnit.createAttachmentUnits.removeBreakSlide' | artemisTranslate }}</span>
         <fa-icon [icon]="faQuestionCircle" [ngbTooltip]="'artemisApp.attachmentUnit.createAttachmentUnits.removeBreakSlideTooltip' | artemisTranslate"></fa-icon>
-        <div *ngIf="removeBreakSlides" class="alert alert-warning mt-3 ml-3">
-            <span>
-                {{ 'artemisApp.attachmentUnit.createAttachmentUnits.removeBreakSlideInfo' | artemisTranslate }}
-            </span>
+        <div *ngIf="removeBreakSlides">
+            <div class="alert alert-warning mt-3 ml-3">
+                <span>
+                    {{ 'artemisApp.attachmentUnit.createAttachmentUnits.removeBreakSlideInfo' | artemisTranslate }}
+                </span>
+            </div>
+            <input type="text" class="form-control" id="removeBreakSlidesKeyword" placeholder="Break" autocomplete="off" [(ngModel)]="removeBreakSlidesKeyword" />
```

Comment:
Ignore if irrelevant: You could use a translation here, but I'm not sure if that's actually more useful or rather annoying for instructors.